buildpack/bp_descriptor.go

```diff
@@ -70,8 +70,6 @@ func ReadBpDescriptor(path string) (*BpDescriptor, error) {
 		for _, stack := range descriptor.Stacks {
 			if stack.ID == "io.buildpacks.stacks.bionic" {
```

Comment:
should we add `jammy` given it's the primary stack that paketo supports?

Reply:
As discussed - we will opt for a len == 1 && stack == bionic check, as bionic is encoded in the spec.